integration/ct_functions.sh

```diff
@@ -0,0 +1,90 @@
+# Functions for setting up CT personalities in Trillian integration tests
+# Requires github.com/google/trillian/integration/functions.sh
+
+declare -a CT_SERVER_PIDS
+CT_SERVERS=
```

Comment:
nit: I have a slight preference for `VAR=''` over `VAR=`. I think it looks less weird. Your call.